example/index-deno.js

```diff
@@ -0,0 +1,31 @@
+import { performance } from 'https://deno.land/std@0.159.0/node/perf_hooks.ts'
+import * as path from 'https://deno.land/std@0.159.0/path/mod.ts'
+import { Resvg } from 'npm:@resvg/resvg-js'
```

Comment:
Thank you for your contribution to deno's support for the Node API, can you help review this PR? I have some doubts:

How do I import a local file of an npm package?

e.g.

```js
import { Resvg } from 'npm:../index'
```